src/V3Ast.h

```diff
@@ -1973,6 +1973,20 @@ class AstNode VL_NOT_FINAL {
     AstNodeDType* findBasicDType(VBasicDTypeKwd kwd) const;
     static AstBasicDType* findInsertSameDType(AstBasicDType* nodep);
 
+    enum class Castable : uint8_t {
+        UNSUPPORTED,
+        SAMEISH,
+        COMPATIBLE,
+        ENUM_EXPLICIT,
+        ENUM_IMPLICIT,
+        DYNAMIC_CLASS,
+        INCOMPATIBLE
+    };
```

Comment:
Please make this a VCastable class similar to VSigning and others, at the file level rather than under AstNode.

Reply:
Done